docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -0,0 +1,812 @@
+---
+title: Pre-packaged jobs
+description: Learn how to set up and use pre-packaged jobs.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+
+Pre-packaged jobs are ready-to-use workflow jobs that help you automate common tasks like building, submitting, and testing your app. They provide a standardized way to handle these operations without having to write custom job configurations from scratch. This guide covers the available pre-packaged jobs and how to use them in your workflows.
+
+## Build
+
+Build your project into an Android or iOS app.
+
+Build jobs can be customized so that you can execute custom commands during the build process. See [Custom builds](/custom-builds/get-started/) for more information.
+
+### Prerequisites
+
+To successfully use the build job, you'll need to complete a build with EAS CLI using the same platform and profile as the pre-packaged job. Learn how to [create your first build](/build/setup/) to get started.
+
+### Syntax
+
+```yaml
+jobs:
+  build_app:
+    type: build
+    params:
+      platform: android | ios # required
+      profile: string # optional, default: production
+```
+
+#### Parameters
+
+You can pass the following parameters into the `params` list:
+
+| Parameter | Type   | Description                                                            |
+| --------- | ------ | ---------------------------------------------------------------------- |
+| platform  | string | Required. The platform to build for. Can be either `android` or `ios`. |
```

Comment:
```suggestion
| platform  | string | **Required.** The platform to build for. Can be either `android` or `ios`. |
```

Let's emphasize it by making it bold so skimming through this doc has less chances for missing it.